### PR TITLE
Increase keepalived log level (can be overwritten)

### PR DESCRIPTION
**Pull request by ndf-dw**
Increase keepalived log level by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Another role or deployment method must be devised to use notification scripts su
 * `keepalived_global_defs`: configure the `global_defs` block ;
 * `keepalived_vrrp_scripts`: configure one or more `vrrp_script` ;
 * `keepalived_vrrp_instances`: configure one or more `vrrp_instance`.
+* `keepalived_default_args`: configure defaults arguments for keepalived daemon (set "--log-detail --log-facility=7" by default)
```
Comment (tleguern) [inline README.md]:
```suggestion
* `keepalived_flags`: flags to pass to the keepalived daemon (set `--log-detail --log-facility=7` by default)
```

File: defaults/main.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ---
 keepalived_install: no
 keepalived_packages: "{{ __keepalived_packages }}"
+keepalived_default_args: "--log-detail --log-facility=7"
```
Comment (tleguern) [inline defaults/main.yml]:
```suggestion
keepalived_flags: "--log-detail --log-facility=7"
```

File: tasks/main.yml [hunk 3]
```diff
@@ -12,6 +12,13 @@
     state: present
   when: keepalived_install
 
+- name: Ensure default keepalived conf is installed
+  template:
+    src: keepalived.default
+    dest: /etc/default/keepalived
+    mode: 0644
+  notify: Restart keepalived
+
```
Comment (tleguern) [inline tasks/main.yml:21]: This is probably Debian only but as I don't have any other installation target this is fine for now.

File: tasks/main.yml [hunk 4]
```diff
@@ -12,6 +12,13 @@
     state: present
   when: keepalived_install
 
+- name: Ensure default keepalived conf is installed
```
Comment (tleguern) [inline tasks/main.yml]:
```suggestion
- name: Ensure keepalived command line arguments are defined.
```

File: templates/keepalived.default [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+# Options to pass to keepalived
+
+# DAEMON_ARGS are appended to the keepalived command-line
+DAEMON_ARGS="{{ keepalived_default_args }}"
```
Comment (tleguern) [inline templates/keepalived.default]:
```suggestion
DAEMON_ARGS="{{ keepalived_flags }}"
```

File: templates/keepalived.default [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+# Options to pass to keepalived
```
Comment (tleguern) [inline templates/keepalived.default:2]: This comment is not really helpful as it is a repetition of the comment on line 4.

**Review by tleguern — CHANGES_REQUESTED** [review]
Hello Nicolas, I added a few suggestions to rename the new option to `keepalived_flags`. Can you take a look?

File: templates/keepalived.default (hunk 6 above)
Comment (ndf-dw) [reply templates/keepalived.default:2]: It comes from the file installed by Debian!
I think they anticipated other variables

File: tasks/main.yml (hunk 3 above)
Comment (ndf-dw) [reply tasks/main.yml:21]: Indeed, didn't find same file on redhat for example.

File: templates/keepalived.default (hunk 6 above)
Comment (tleguern) [reply templates/keepalived.default:2]: Ok, we can keep it then.

**Review by tleguern — APPROVED** [review]

**Comment by tleguern** [comment]
Thank you Nicolas ;)

**Pull request merged**